### ISSUE TITLE
Question about Skipgram

### ISSUE THREAD

**Issue by Fujiki-Nakamura** [Fujiki-Nakamura/tutorial_nlp#3]
- Skipgram with Negative Sampling では logprob を sigmoid(logit) として計算している。ただの Skipgram でもそれでよいのではないか？softmax(logit) とすると、予測対象の周辺単語の確率を全て足すと 1 という制約が入る気がする。
  - とはいっても、いずれにせよ、上げたい対数尤度は問題なく上がるように最適化されるので、やりたいことに変わりはない。
